README.md

```diff
@@ -38,3 +38,17 @@ With the world and copter in place, launch cartographer to generate SLAM:
 ```bash
 ros2 launch ardupilot_ros cartographer.launch.py
 ```
+
+If you'd like to get the information from Cartographer to go into Ardupilot's extended kalman filter, you will need to change some parameters, you can do that through any GCS, including mavproxy:
+
+-  AHRS_EKF_TYPE = 3 to use EKF3
+-  EK2_ENABLE = 0 to disable EKF2
+-  EK3_ENABLE = 1 to enable EKF3
+-  EK3_SRC1_POSXY = 6 to set position horizontal source to ExternalNAV
+-  EK3_SRC1_POSZ = 1 to set position vertical source to Baro
+-  EK3_SRC1_VELXY = 6 to set velocity horizontal source to ExternalNAV
+-  EK3_SRC1_VELZ = 6 to set vertical velocity source to ExternalNAV
+-  EK3_SRC1_YAW = 6 to set yaw source to ExternalNAV
+-  GPS_TYPE = 0 to disable the GPS
```

Comment:
why disable the GPS? On real vehicles it would be very helpful for log analysis, even if not actually used in flight

Reply:
Fair point! I will change it here and in ArduPilot/ardupilot_wiki#5366

Reply:
Agree with Tridge that normally users, particularly researches, would have GPS available and have it as a source of truth.